Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs

```diff
@@ -331,15 +331,25 @@ private void TrackVRT()
                             mAct.Error += $"Differences from baseline was found.{System.Environment.NewLine}{result.DiffUrl}";
 
                             //Add difference image to act screenshots
-                            if(result.DiffUrl != null){
-                                General.DownloadImage($"{WorkSpace.Instance.Solution.VRTConfiguration.ApiUrl}/{ Path.GetFileName(result.DiffUrl)}", mAct, true, "Difference_Image");
+                            if(result.DiffUrl != null)
+                            {
+                                string DiffrenceImage = General.DownloadImage($"{WorkSpace.Instance.Solution.VRTConfiguration.ApiUrl}/{ Path.GetFileName(result.DiffUrl)}", mAct);
+                                if(!string.IsNullOrEmpty(DiffrenceImage) && File.Exists(DiffrenceImage))
+                                {
+                                    Act.AddArtifactToAction("Difference_Image", mAct, DiffrenceImage);
+                                }                                
                             }
                             
 
                             //Add baseline image to act screenshots
                             if(result.BaselineUrl != null)
                             {
-                                General.DownloadImage($"{WorkSpace.Instance.Solution.VRTConfiguration.ApiUrl}/{Path.GetFileName(result.BaselineUrl)}", mAct, true, "BaseLine_Image");
+                                mAct.previewBaselineImageName = Path.GetFileName(result.BaselineUrl);
+                                string BaseLineImage = General.DownloadImage($"{WorkSpace.Instance.Solution.VRTConfiguration.ApiUrl}/{Path.GetFileName(result.BaselineUrl)}", mAct);
```

Comment:
use Path.DirectorySepeator char to support it on Linux

Reply:
This is not file path, we are configuring URL.